db/migrate/20200210194758_add_first_time_coaching_to_coach_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddFirstTimeCoachingToCoachApplications < ActiveRecord::Migration[5.2]
+  def change
+    add_column :coach_applications, :first_time_coaching, :boolean
```

Comment:
Since this is already deployed and we already have the table working (even if not being used for real), we should make sure that this column is going to populated with data for all the existing data.

As such, we should add a `default: false` at the end.